app/templates/components/clear-button.hbs

```diff
@@ -1,6 +1,4 @@
-<svg width="16px" height="16px" viewBox="0 0 16 16" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
-  <g class="svg-stroke" transform="translate(3.000000, 3.7500000)" stroke="#000000" stroke-width="2" fill="none" fill-rule="evenodd">
-    <circle cx="5.5" cy="5.5" r="5.5"></circle>
-    <path d="M1.98253524,1.98253524 L9,9" id="Line" stroke-linecap="square"></path>
-  </g>
+<svg xmlns="http://www.w3.org/2000/svg" width="13" height="13">
```

Comment:
Would it perhaps be better to split this out into its own svg file, and then use it in the component? Perhaps with https://github.com/ivanvotti/ember-svg-jar?